src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+tile.natura.tiger_fence.name=Tigerholzzaun
+tile.natura.tiger_fence_gate.name=Tigerholzzauntor
+
+tile.natura.willow_button.name=Weidenholzknopf
+tile.natura.willow_pressure_plate.name=Weidenholzdruckplatte
+tile.natura.willow_trap_door.name=Weidenholzfalltür
+tile.natura.willow_fence.name=Weidenholzzaun
+tile.natura.willow_fence_gate.name=Weidenholzzauntor
+
+tile.natura.eucalyptus_button.name=Eukalyptusholzknopf
+tile.natura.eucalyptus_pressure_plate.name=Eukalyptusholzdruckplatte
+tile.natura.eucalyptus_trap_door.name=Eukalyptusholzfalltür
+tile.natura.eucalyptus_fence.name=Eukalyptusholzzaun
+tile.natura.eucalyptus_fence_gate.name=Eukalyptusholzzauntor
+
+tile.natura.hopseed_button.name=Hopfensaatknopf
+tile.natura.hopseed_pressure_plate.name=Hopfensaatdruckplatte
+tile.natura.hopseed_trap_door.name=Hopfensaatfalltür
+tile.natura.hopseed_fence.name=Hopfensaatzaun
+tile.natura.hopseed_fence_gate.name=Hopfensaatzauntor
+
+tile.natura.sakura_button.name=Sakuraholzknopf
+tile.natura.sakura_pressure_plate.name=Sakuraholzdruckplatte
+tile.natura.sakura_trap_door.name=Sakuraholzfalltür
+tile.natura.sakura_fence.name=Sakuraholzzaun
+tile.natura.sakura_fence_gate.name=Sakuraholzzauntor
+
+tile.natura.redwood_button.name=Rotholzknopf
+tile.natura.redwood_pressure_plate.name=Rotholzdruckplatte
+tile.natura.redwood_trap_door.name=Rotholzfalltür
+tile.natura.redwood_fence.name=Rotholzzaun
+tile.natura.redwood_fence_gate.name=Rotholzzauntor
+
+tile.natura.overworld_bookshelves.maple.name=Ahornholzbücherregal
+tile.natura.overworld_bookshelves.silverbell.name=Silberglockenbücherregal
+tile.natura.overworld_bookshelves.amaranth.name=Amaranthbücherregal
+tile.natura.overworld_bookshelves.tiger.name=Tigerholzbücherregal
+tile.natura.overworld_bookshelves.willow.name=Weidenholzbücherregal
+tile.natura.overworld_bookshelves.eucalyptus.name=Eukalyptusholzbücherregal
+tile.natura.overworld_bookshelves.hopseed.name=Hopfensaatbücherregal
+tile.natura.overworld_bookshelves.sakura.name=Sakuraholzbücherregal
+tile.natura.overworld_bookshelves.redwood.name=Rotholzbücherregal
+
+tile.natura.overworld_workbenches.maple.name=Ahornholzwerkbank
+tile.natura.overworld_workbenches.silverbell.name=Silberglockenwerkbank
+tile.natura.overworld_workbenches.amaranth.name=Amaranthwerkbank
+tile.natura.overworld_workbenches.tiger.name=Tigerholzwerkbank
+tile.natura.overworld_workbenches.willow.name=Weidenholzwerkbank
+tile.natura.overworld_workbenches.eucalyptus.name=Eukalyptusholzwerkbank
+tile.natura.overworld_workbenches.hopseed.name=Hopfensaatwerkbank
+tile.natura.overworld_workbenches.sakura.name=Sakuraholzwerkbank
+tile.natura.overworld_workbenches.redwood.name=Rotholzwerkbank
+
+#######################
+#    Nether Blocks    #
+#######################
+
+tile.natura.nether_logs.ghostwood.name=Geisterholz
+tile.natura.nether_logs.ghostwood.tooltip=Netherbaum\nBleich wie ein Gespenst
+tile.natura.nether_logs.darkwood.name=Dunkelholz
```

Comment:
Maybe "Düsterholz" or "Finsterholz" would be a cool name.

Reply:
Good idea, I like "Finsterholz" a bit more I think. :)